WcaOnRails/lib/results_validators/persons_validator.rb

```diff
@@ -19,6 +19,8 @@ class PersonsValidator < GenericValidator
     NON_MATCHING_NAME_WARNING = "The name '%{name}' provided for %{wca_id} does not match the current record in the WCA database ('%{expected_name}'). "
     NON_MATCHING_COUNTRY_WARNING = "The country '%{country}' provided for %{name} (%{wca_id}) does not match the current record in the WCA database ('%{expected_country}')."\
     " If this is an error, fix it. Otherwise, leave a comment to the WRT about it."
+    MULTIPLE_NEWCOMERS_WITH_SAME_NAME_ERROR = "There are multiple new competitors with the exact same name: %{name}. Due to a bug in the code, we cannot upload the Results. Please contact the Results Team with your JSON (free of all other errors) to help upload the results."
```

Comment:
Proposal: "There are multiple new competitors with the exact same name: %{name}. Due to a limitation in the current posting procedure, this will prevent WRT from uploading the results. Please temporarily rename all but one of these newcomers (e.g. by adding numbers) and explain what you did in your submission mail, so that WRT will reinstate the original names after uploading the results."

Reply:
I've made this change with one wording change and an addition to contact WRT if they need any help. Usually this only happens to Vietnamese competitions, and those Delegates know how to handle this, but in the very rare chance it happens to someone else, they may be very confused :P